pkg/apis/kops/validation/validation.go

```diff
@@ -1353,6 +1353,12 @@ func validateNetworkingCilium(cluster *kops.Cluster, v *kops.CiliumNetworkingSpe
 		}
 	}
 
+	if v.Ingress != nil && fi.ValueOf(v.Ingress.Enabled) {
+		if v.Ingress.DefaultLoadBalancerMode != "" {
+			allErrs = append(allErrs, IsValidValue(fldPath.Child("ingress", "defaultLoadBalancerMode"), &v.Ingress.DefaultLoadBalancerMode, []string{"shared", "dedicated"})...)
+		}
+	}
```

Comment:
Is it allowed to set SharedLoadBalancerServiceName if the mode is dedicated?

Reply:
Might make sense to completely remove this option. The cilium helm chart does not make the name configurable. Wdyt?

Reply:
At least from the helm chart, I see that it is always set and does not depend on the mode:

https://github.com/cilium/cilium/blob/main/install/kubernetes/cilium/templates/cilium-configmap.yaml#L248